classes/data-stores/ActionScheduler_DBStore.php

```diff
@@ -622,7 +629,7 @@ protected function generate_claim_id() {
 	 * @param int       $limit Number of action to include in claim.
 	 * @param \DateTime $before_date Should use UTC timezone.
 	 * @param array     $hooks Hooks to filter for.
-	 * @param string    $group Group to filter for.
+	 * @param string    $group Group to filter for, adding IGNORE_GROUP_FLAG in front of the group will have it ignored.
```

Comment:
Would this be more accurate?

```suggestion
	 * @param string    $group Comma-separated list of groups to filter by. Adding IGNORE_GROUP_FLAG in front of a group will cause it to be ignored.
```